src/plugins/themes.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+const fs = require("fs");
+const Helper = require("../helper");
+const requireg = require("requireg");
+const colors = require("colors/safe");
+const path = require("path");
+const themes = new Map();
+
+module.exports = {
+	get: get,
+	fileName: fileName
+};
+
+fs.readdir("client/themes/", (err, dirThemes) => {
+	if (err) {
+		return;
+	}
+	dirThemes
+		.filter((theme) => theme.endsWith(".css"))
+		.map(makeLocalThemeObject)
+		.forEach((theme) => themes.set(theme.name, theme));
+});
+
+Helper.config.plugins
+	.map(getModuleInfo)
+	.filter((module) => module !== undefined)
```

Comment:
You're iterating same array 6 times here, I understand ES6 is nice, but this is kinda silly.